glyphs-reader/src/font.rs

```diff
@@ -1971,7 +1973,22 @@ impl Instance {
             value
                 .width_class
                 .as_ref()
-                .map(|v| f64::from_str(v).unwrap())
+                .map(|v| {
+                    match i32::from_str(v).unwrap() {
+                        1 => 50.0,  // Ultra-condensed
+                        2 => 62.5,  // Extra-condensed
+                        3 => 75.0,  // Condensed
+                        4 => 87.5,  // Semi-condensed
+                        5 => 100.0, // Medium (normal)
+                        6 => 112.5, // Semi-expanded
+                        7 => 125.0, // Expanded
+                        8 => 150.0, // Extra-expanded
+                        9 => 200.0, // Ultra-expanded
+                        _ => {
+                            panic!("Unsupported instance width_class {}", v);
```

Comment:
glyphsLib also raises NotImplementedError in this situation, which is unlikely enough to warrant an error, not just a warning. The Glyphs Python scripting API docs say this field only has int values from 1 to 9 corresponding to OS/2 usWidthClass.